layers/+vim/programming/config.vim

```diff
@@ -157,3 +157,14 @@ augroup END
     " Reset errorformat to its default value for cooperating with asyncrun.vim
     autocmd BufEnter * set errorformat&
 " }
+
+" vim-swoop {
+    " Do not let vim-swoop create keybindings
+    let g:swoopUseDefaultKeyMap = 0
+    " Visual selection mode mapping: swoop current selection.
+    vmap <Leader>s :call SwoopMultiSelection()<CR>
+    " Normal mode mapping: <Leader>s to start swoop buffer.
+    nnoremap <Leader>s :call SwoopMulti()<CR>
+    " normal mode mapping: select current word, then swoop.
+    nnoremap <Leader>ss viw:call SwoopMultiSelection()<CR>
```

Comment:
`<Leader>s` and `<Leader>ss` could result in a mapping delay. When you type `<Leader>s`, vim will judge whether what you want to input is `<Leader>s` or actually `<Leader>ss`. See `:h timeoutlen`.